src/ui/server/controllers/insurance/policy-and-export/type-of-policy/index.ts

```diff
@@ -97,6 +99,10 @@ export const post = async (req: Request, res: Response) => {
       return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`);
     }
 
+    if (isCheckAndChangeRoute(req.originalUrl)) {
```

Comment:
For this change and the previous files, do we not need to update the unit test? What does the coverage say?

Reply:
forgot to do this